get_ocp3_oa.yml

```diff
@@ -7,3 +7,17 @@
         dest: "{{ workspace }}/openshift-ansible"
         version: "release-{{ (lookup('env', 'OPENSHIFT_VERSION') or 'v3.11')[1:] }}"
         force: yes
+
+    - name: Creating a fake file for OCP-3.7
+      lineinfile:
+        path: "{{ lookup('env', 'WORKSPACE') or '../'}}/openshift-ansible/playbooks/deploy_cluster.yml"
```

Comment:
You can use here the `workspace` variable, as it is used on line 7. Same thing for line 20.

Reply:
done